docs/checkers/index.md

```diff
@@ -1657,6 +1657,32 @@ class CashRegister:
 - [R0201](#R0201)
 - [StackOverflow: What is the difference between `@staticmethod` and `@classmethod` in Python?]
 
+(W0245)=
+
+### Super without brackets (W0245)
```

Comment:
I think a better place for this method is immediately after the entry for "Bad parent init".